src/lib/module-info.ts

```diff
@@ -140,9 +142,14 @@ function withoutExtension(str: string, ext: string): string {
 }
 
 /** Returns a map from filename (path relative to `directory`) to the SourceFile we parsed for it. */
-async function allReferencedFiles(entryFilenames: ReadonlyArray<string>, fs: FS, baseDirectory: string): Promise<Map<string, ts.SourceFile>> {
+export async function allReferencedFiles(
```

Comment:
allReferencedFiles now scans through .d.ts AND .ts files, instead of only looking at d.ts files. It uses extension d.ts-vs-ts to decide whether the referenced file is a type file or a test file.

This is not precisely equivalent to the old algorithm but it produces the same results on DT (so far at least).